scripts/vim-patch.sh

```diff
@@ -30,9 +30,9 @@ usage() {
   echo "    -g {vim-revision}  Download a Vim patch."
   echo "    -s                 Create a vim-patch pull request."
   echo "    -r {pr-number}     Review a vim-patch pull request."
-  echo '    -V                 Clone the Vim source code to $VIM_SOURCE_DIR.'
+  echo "    -V                 Clone the Vim source code to \$VIM_SOURCE_DIR."
   echo
-  echo '    $VIM_SOURCE_DIR controls where Vim sources are found'
+  echo "    \$VIM_SOURCE_DIR controls where Vim sources are found"
```

Comment:
this is strange, shellcheck prefers double quotes?

Reply:
No, but it does not warn about `$FOO` not being expanded then (in single quotes).. ;)